include/umf/providers/provider_os_memory.h

```diff
@@ -57,12 +57,23 @@ typedef enum umf_numa_mode_t {
     /// fulfilled, memory will be allocated from other nodes.
     UMF_NUMA_MODE_PREFERRED,
 
+    /// Allocation will be split evenly across nodes specified in nodemask.
+    /// An extra argument could be passed in params structure to specify other distribution.
```

Comment:
do you mean partitions arguments in `umf_os_memory_provider_params_t`? these are not `an extra argument` - you could be more precise, pls

Reply:
done